dpdata/plugins/orca.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np
+
+from dpdata.format import Format
+from dpdata.orca.output import read_orca_sp_output
+from dpdata.unit import EnergyConversion, ForceConversion
+
+energy_convert = EnergyConversion("hartree", "eV").value()
+force_convert = ForceConversion("hartree/bohr", "eV/angstrom").value()
+
+
+@Format.register("orca/spout")
+class ORCASPOutFormat(Format):
+    """ORCA single point energy output.
+
+    Note that both the energy and the gradient should be
+    printed into the output file.
+    """
+
+    def from_labeled_system(self, file_name: str, **kwargs) -> dict:
+        """Read from ORCA single point energy output.
+
+        Parameters
+        ----------
+        file_name : str
+            file name
+        **kwargs
+            keyword arguments
+
+        Returns
+        -------
+        dict
+            system data
+        """
+        symbols, coord, energy, forces = read_orca_sp_output(file_name)

```

Comment:
Consider adding error handling for the case where the file cannot be read or parsed correctly by the `read_orca_sp_output` function. This can improve the robustness of your code. [important]